src/main/java/Katleen.java

```diff
@@ -0,0 +1,207 @@
+   private static final Path filePath = Paths.get(filename);
+   private static void loadFileContents() {
+       try {
+           File file = filePath.toFile();
+           if (!file.exists()) {
+               System.out.println("Savefile not found, creating new file");
+               file = new File("./" + filename);
+               System.out.println("File created at: " + file.getAbsolutePath());
+           } else {
+               Scanner s = new Scanner(filePath);
+               int temp = 0;
+               while (s.hasNext()) {
+                   String savedTask = s.nextLine();
+                   String[] save = parseCommand(savedTask);
+                   String[] splitSaved = save[0].split(" ");
+                   Task saved = getTask(savedTask, splitSaved[0], splitSaved, false);
+                   tasks.add(saved);
+                   if (!save[1].isEmpty()) {
+                       tasks.get(temp).setDone(true);
+                   }
+                   temp++;
+               }
+               }
+       } catch (IncompleteTaskException e) {
+           System.out.println("Could not create new task");
+       } catch (IOException e) {
+           System.out.println("IOException");
+       }
+   }
+
+    public static void main(String[] args) throws IncompleteTaskException, UnrecognizedCommandException {
+        System.out.println(LINE);
+        System.out.println("Hello! I'm Katleen.");
+        System.out.println("What can I do for you?");
+        System.out.println(LINE);
+        loadFileContents();
+        Scanner in = new Scanner(System.in);
+        String text = "";
+        while (!text.equals("bye")) {
+            text = in.nextLine();
+            System.out.println(LINE);
+            parseString(text);
+            System.out.println(LINE);
+        }
+    }
+
+    private static void parseString(String input) {
+        String[] splitInput = input.split(" ");
+        String cmdWord = splitInput[0];
+        int taskCount = tasks.size();
+        try {
+            switch (cmdWord) {
+            case "list":
+                if (tasks.isEmpty()) {
+                    System.out.println("You have no tasks!");
+                    break;
+                }
+                System.out.println("Here are the tasks in your list:");
+                int count = 1;
+                for (Task t : tasks) {
+                    System.out.print(count + ". ");
+                    t.printTask();
+                    count++;
+                }
+                break;
+            case "bye":
+                for (Task t : tasks) {
+                    FileWriter fw = new FileWriter(filename);
+                    fw.write(t.toString());
+                }
+                System.out.println(LINE);
+                System.out.println("Bye, have a nice day!");
+                System.out.println(LINE);
+                break;
+            case "mark":
+            case "unmark":
```

Comment:
add //Fallthrough comment if there is empty case statements. See the coding standards for more info